fix(minimal): pre-seal a first block to trigger maintain

iulianbarbu opened the pull request:
# Description

After making fork aware txpool the default, instant seal of minimal node stopped working as expected because any transaction sent to it got stuck in mempool , since there are no active views to include the tx in.

To overcome this we can create a first view by pre-sealing a first empty block, which triggers the `maintain` phase and view building logic. This is compatible with single-state tx pool too.

## Integration

N/A

## Review Notes

N/A

skunert commented:
Ah wait, I think we should trigger this block building from the minimal node and not from within substrate.

iulianbarbu commented:
Ok, moved the fix into minimal-node and removed the prdoc (I believe it is not necessary since we don't release anything in relation to minimal-template-node).

skunert commented:
I advised Iulian to implement this fix as a quick fix. But true we should fix the root cause 👍
